### PR TITLE
Added klongpy as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes I think this cannot be noarch due to scripts? Not sure, happy to change to noarch otherwise.

**Comment by thewchan** [comment]
@conda-forge/staged-recipes pinging for awareness but also let me know if it's okay to change this to noarch.

File: recipes/klongpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/klongpy-{{ version }}.tar.gz
+  sha256: be4981b8621fc5eb3d74948404e8272830e4da3eb52c800b632fb52ad0e0afdc
+
+build:
+  skip: true  # [py<38]
```
Comment (ocefpaf) [inline recipes/klongpy/meta.yaml:13]: Can you add a comment that this one has scripts and cannot be noarch? Or, better yet, use the "semi-arch" new syntax and make it a win and nix noarch package?

Comment (thewchan) [reply recipes/klongpy/meta.yaml:13]: @ocepaf can you remind me where the documentation of this semi-arch syntax might be?

Comment (ocefpaf) [reply recipes/klongpy/meta.yaml:13]: You are only skipping Python, right? So why not vanilla noarch?

Comment (ocefpaf) [reply recipes/klongpy/meta.yaml:13]: PS: here is an example of a Windows only noarch: https://github.com/jaimergp/win_inet_pton-feedstock/blob/209342dfb957b8aafc8094f703849bc3becad7a2/recipe/meta.yaml

Comment (thewchan) [reply recipes/klongpy/meta.yaml:13]: @ocefpaf Still a little confused, I'm thinking this has scripts and therefore cannot be noarch, but would adding `__win` to dependency make this work as noarch even though it has scripts? Happy to look at documentations on Conda about this if it exists

Comment (ocefpaf) [reply recipes/klongpy/meta.yaml:13]: Ah. That was the reason. I forget. To make it noarch for python for 1 package for each platform is a bit more involved. Let's merge this and we can work on it later in the feedstock.

Comment (thewchan) [reply recipes/klongpy/meta.yaml:13]: Appreciate it @ocepaf I'll ping you in the repo once it materializes

**Pull request merged**